src/Symfony/Component/HttpFoundation/RequestMatcher.php

```diff
@@ -89,6 +98,18 @@ public function matchHost($regexp)
         $this->host = $regexp;
     }
 
+    /**
+     * Adds a check for the the URL port.
+     *
+     * @param int|null $port The port number to connect to
+     */
+    public function matchPort($port)
```

Comment:
this one should use `int $port = null`, the existing public methods of this class have been added before php scalar typehints and cannot be changed for BC